src/component/graphic/GraphicView.ts

```diff
@@ -441,10 +441,17 @@ function updateCommonAttrs(
 ) {
     if (!el.isGroup) {
         const elDisplayable = el as Displayable;
+        elDisplayable.cursor = (elOption as GraphicComponentDisplayableOption).cursor;
```

Comment:
In zrender displayable has a default cursor value `'pointer'`. I think we should keep it here. Otherwise the undefined value will be defaulted to `default` in https://github.com/ecomfe/zrender/blob/master/src/dom/HandlerProxy.ts#L604

Reply:
Not sure if it's better to use the default value from `Displayable.prototype.cursor` than the string `cursor`. One good side is that it always keeps uniform value with zrender.